pkgs/applications/misc/jquake/default.nix

```diff
@@ -0,0 +1,72 @@
+{ lib, stdenv, fetchurl, copyDesktopItems, makeDesktopItem, unzip, jre8 }:
+
+stdenv.mkDerivation rec {
+  pname = "jquake";
+  version = "1.6.1";
+
+  src = fetchurl {
+    url = "https://fleneindre.github.io/downloads/JQuake_${version}_linux.zip";
+    sha256 = "0nw6xjc3i1b8rk15arc5d0ji2bycc40rz044qd03vzxvh0h8yvgl";
+  };
+
+  desktopFile = makeDesktopItem {
+    name = "JQuake";
+    desktopName = "JQuake";
+    exec = "JQuake";
+    comment = "Real-time earthquake map of Japan";
+  };
+
+  nativeBuildInputs = [ unzip copyDesktopItems ];
+
+  sourceRoot = ".";
+
+  patchPhase = ''
+    runHook prePatch
+
+    # JQuake emits a lot of debug-like messages in console, but I
+    # don't think it's in our interest to void them by default. Log them at
+    # the appropriate level.
+    sed -i "/^java/ s/$/\ | logger -p user.debug/" JQuake.sh
+
+    substituteInPlace JQuake.sh \
+      --replace "java -jar " "exec ${jre8.outPath}/bin/java -jar $out/lib/"
+
+    # By default, an 'errors.log' file is created in the current directory.
+    # cd into a temporary directory and let it be created there.
+    substituteInPlace JQuake.sh \
+      --replace "[JAR FOLDER]" "\$(mktemp -p /tmp -d jquake-errlog-XXX)"
+
+    runHook postPatch
+  '';
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    chmod +x JQuake.sh
+
+    mkdir -p $out/{bin,lib}
+    mv JQuake.sh $out/bin/JQuake
+    mv {JQuake.jar,JQuake_lib} $out/lib
+    mv sounds $out/lib
+
+    mkdir -p $out/share/licenses
+    mv LICENSE* $out/share/licenses
```

Comment:
This probably collides with other packages.

Reply:
I didn't know where a more appropriate place could have been. Sorry, I never packaged something before.

Reply:
```suggestion
    mkdir -p $out/share/licenses
    mv LICENSE* $out/share/licenses/jquake
```
yes please.